mix.exs

```diff
@@ -73,10 +73,9 @@ defmodule Membrane.RTP.Plugin.MixProject do
 
   defp deps do
     [
-      {:membrane_core,
-       github: "membraneframework/membrane_core", tag: "v0.9.0-rc.0", override: true},
+      {:membrane_core, "~> 0.9.0", override: true},
       {:membrane_rtp_format, "~> 0.3.1"},
-      {:membrane_rtp_h264_plugin, "~> 0.8.0"},
+      {:membrane_rtp_h264_plugin, "~> 0.8.0", only: :test},
```

Comment:
I usually removed the last number from test only deps as they won't be published and locally `mix.lock` takes care of ensuring the proper version - here it would allow updating h264_ffmpeg to version compatible with new core